benchmark_runner/main/environment_variables.py

```diff
@@ -31,6 +31,23 @@ def __init__(self):
         self._environment_variables_dict['elasticsearch'] = os.environ.get('ELASTICSEARCH', '')
         self._environment_variables_dict['elasticsearch_port'] = os.environ.get('ELASTICSEARCH_PORT', '')
 
+        # This parameter 'True'/'False' for debug workload, when its set to 'True' it will stop the workload when finishing
+        self._environment_variables_dict['stop_when_workload_finish'] = os.environ.get('STOP_WHEN_WORKLOAD_FINISH', '')
+
+        # default parameter - change only if needed
+        # Parameters below related to 'run_workload()'
+        self._environment_variables_dict['workloads'] = ['stressng_pod', 'stressng_vm', 'uperf_pod', 'uperf_vm', 'hammerdb_pod_mariadb', 'hammerdb_vm_mariadb',  'hammerdb_pod_postgres', 'hammerdb_vm_postgres', 'hammerdb_pod_mssql', 'hammerdb_vm_mssql']
+        self._environment_variables_dict['namespace'] = os.environ.get('NAMESPACE', 'benchmark-operator')
+        # run Hammerdb workload with ocs pvc True/False
+        self._environment_variables_dict['ocs_pvc'] = os.environ.get('OCS_PVC', 'True')
```

Comment:
Whereas this is a configuration option that it is meaningful for the user to set.

Reply:
There are defaults parameters for the CI runs but the user can change it for his needs.

Reply:
User is not able to change this file at all. User needs just pass environment variable when running the container.
This file is for internal debugging